x-pack/plugins/security_solution/cypress/cypress_serverless.config.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { defineCypressConfig } from '@kbn/cypress-config';
+import { esArchiver } from './support/es_archiver';
+
+// eslint-disable-next-line import/no-default-export
+export default defineCypressConfig({
+  defaultCommandTimeout: 60000,
+  execTimeout: 60000,
+  pageLoadTimeout: 60000,
+  responseTimeout: 60000,
+  screenshotsFolder: '../../../target/kibana-security-solution/cypress/screenshots',
+  trashAssetsBeforeRuns: false,
+  video: false,
+  videosFolder: '../../../target/kibana-security-solution/cypress/videos',
+  viewportHeight: 946,
+  viewportWidth: 1680,
+  numTestsKeptInMemory: 10,
+  env: {
+    grepFilterSpecs: true,
+    grepTags: '@serverless --@brokenInServerless',
```

Comment:
We can even exclude tags -- that's awesome!

Reply:
this looks awesome 👍 would there also be a way to tag specific tests (`it` instead of top level describe) ?

Reply:
Yes! You can tag on any level.